tests/flytekit/unit/models/core/test_workflow.py

```diff
@@ -29,7 +29,7 @@ def test_alias():
 
 
 def test_workflow_metadata():
-    obj = _workflow.WorkflowMetadata()
+    obj = _workflow.WorkflowMetadata(queuing_budget=timedelta(seconds=10))
```

Comment:
can you add another test that does the whole .from_flyte_idl(blah.to_flyte_idl()) tests?  there should be some in this file.  Can you test both the case where the metadata is None and when it's not?

Reply:
Done.